NATI covers several gx fact types (#319)
FamilySearch/GEDCOM: attribute-event-requests/INDI-attribute.md

| | | [Affiliation](#affiliation) | |
| `CAST` | 5.0 | [Caste](#caste) | |
| \*     | 5.0 | [Childless](#childless) | encoded as `NCH 0` |
| `NATI` | 5.4 | [Clan](#clan) | one of several types of `NATI` |
| \* | 3.0 | [Died Before Eight](#died-before-eight) | encoded as `DEAT`.`AGE <8y` |
| `EDUC` | 4.0 | Education | |
| `NATI` | 5.4 | [Ethnicity](#ethnicity) | one of several types of `NATI` |

> | `NATI` | 5.4 | [Heimat](#heimat) | one of several types of `NATI` |

Author: ```suggestion
| | | [Heimat](#heimat) | |
```

The most closely related structures are:

> - `NATI`: Defined as "an individual’s national heritage or origin, or other folk, house, kindred, lineage, or tribal interest." While "heimat" is not specifically included, it is related to "origin" and "house" which are.

Author: ```suggestion
- `NATI`: Defined as "an individual’s national heritage or origin, or other folk, house, kindred, lineage, or tribal interest." By contrast "heimat" refers to a geographical region a person is from. That could overlap with a national or other regional identity, but need not do so in general.
```

Related proposals include

- Ethnicity: ethnicities typically have some geographic association, but generally much larger and less fine-grained than a heimat

> - Nationality: generally broader than heimat in what region it covers

Author: ```suggestion
- Nationality: nations are often larger regions than heimats, and may refer to nations that a person belongs to but is not geographically from
```
